File: dotNet/Semantria.Com/Mapping/Output/DocIntention.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "intention")]
    [XmlRootAttribute("intention", Namespace = "")]
    public class DocIntention
    {
        [DataMember(Name = "type")]
        [XmlElementAttribute("type")]
        public string Type { get; set; }

        [DataMember(Name = "evidence_phrase")]
        [XmlElementAttribute("evidence_phrase")]
        public string EvidencePhrase { get; set; }

        [DataMember(Name = "what")]
        [XmlElementAttribute("what")]
        public string What { get; set; }

        [DataMember(Name = "who")]
        [XmlElementAttribute("who")]
        public string Who { get; set; }
    }
}

File: Java/src/main/java/com/semantria/mapping/output/DocIntention.java
package com.semantria.mapping.output;

import javax.xml.bind.annotation.XmlElement;

public final class DocIntention
{
	private String type = null;
	private String evidence_phrase = null;
	private String what = null;
    private String who = null;

	@XmlElement(name="type")
	public String getType() { return type; }
	@XmlElement(name="evidence_phrase")
	public String getEvidencePhrase() { return evidence_phrase; }
	@XmlElement(name="what")
	public String getWhat() { return what; }
    @XmlElement(name="who")
    public String getWho() { return who; }

	public void setType(String type) { this.type = type; }
	public void setEvidencePhrase(String evidence_phrase) { this.evidence_phrase = evidence_phrase; }
	public void setWhat(String what) { this.what = what; }
	public void setWho(String who) { this.who = who; }
}
